documentation/modules/about-cold-warm-migration.adoc

```diff
@@ -29,10 +29,10 @@ The VM disks are copied incrementally using link:https://kb.vmware.com/s/article
 ====
 You must enable CBT on the source VMs and the VM disks.
 
-A VM can support up to 28 CBT snapshots. If that limit is exceeded, a `warm import retry limit reached` error message is displayed. If the VM has preexisting CBT snapshots, it will reach this limit sooner.
+A VM can support up to 28 CBT snapshots. If the source VM has too many CBT snapshots and the Migration Controller is not able to create a new snapshot, warm migration might fail. The Migration Controller deletes each snapshot it creates after it is finished with it.
 ====
```

Comment:
Not 100% clear to me:
 The Migration Controller deletes each snapshot it creates after the snapshot is finished with the controller?
 OR
  The Migration Controller deletes each snapshot it creates after the controller finishes with the snapshot?

Reply:
I do not see how the text can be interpreted in the first way. The subject of both verbs is the MC. But I don't mind repeating the subject. The sentence has too many pronouns.